Tests/PackageLoading/ConventionTests.swift

```diff
@@ -159,21 +158,42 @@ private extension FileSystem {
     }
 }
 
-/// Loads a package using PackageBuilder at the given path. If package description is provided it is used,
-/// otherwise a default package with name of directory is passed to manifest.
+/// A convenience enum for package builder tester. Since most of our convention tests
+/// only requires a Package object with the name filled in.
+enum PackageOrName {
```

Comment:
hmm I guess we only need it when we need to check for excluded paths and target dependencies